hybrasyl/Scripting/HybrasylWorld.cs

```diff
@@ -108,6 +107,73 @@ public HybrasylDialog NewDialog(string displayText, string callback = null)
             return new HybrasylDialog(dialog);
         }
 
+        /// <summary>
+        /// Create a new dialog sequence consisting of a bunch of simple text dialogs.
+        /// </summary>
+        /// <param name="sequenceName">The name of the constructed sequence.</param>
+        /// <param name="textList">A string array of dialog lines that will be used to construct each dialog in the sequence.</param>
+        /// <returns>The constructed dialog seqeunce</returns>
+        public HybrasylDialogSequence NewSimpleDialogSequence(string sequenceName, params string[] textList)
+        {
+            var sequence = new DialogSequence(sequenceName);
+            foreach (var entry in textList)
+            {
+                sequence.AddDialog(new SimpleDialog(entry));
+            }
+            return new HybrasylDialogSequence(sequence);
+        }
+
+        /// <summary>
+        /// Create a new dialog sequence consisting of a simple dialog and a jump to a new sequence. Useful 
+        /// for a lot of dialogs where you need to display one dialog and go back to the main menu.
+        /// </summary>
+        /// <param name="simpleDialog">Text for the simple dialog.</param>
+        /// <param name="jumpTarget">The new sequence to start after the user hits next on the simple dialog.</param>
+        /// <param name="callback">An optional Lua callback expression that will be attached to the simple dialog.</param>
+        /// <param name="name">An optional name to give the dialog sequence.</param>
+        /// <returns>The constructed dialog sequence</returns>
+        public HybrasylDialogSequence NewTextAndJumpDialog(string simpleDialog, string jumpTarget, string callback = "", string name = null)
+        {
+            DialogSequence sequence;
+            if (name == null)
+                sequence = new DialogSequence(Guid.NewGuid().ToString());
```

Comment:
Are guid's static across restarts?

Reply:
No, but, that's irrelevant. `DialogSequences` are created at script runtime and only exist for the lifetime of the server. If you want to refer to a sequence again later, you provide a name for it in Lua as a function parameter; otherwise it is given a random string name.